docs/development/running_tests.rst

```diff
@@ -65,15 +65,15 @@ TARDIS prints out the progress:
 
 .. code-block:: shell
 
-    > python setup.py test --args="--integration=integration.yml -m integration
+    > python setup.py test --args="--integration-tests=integration.yml -m integration
     --generate-reference --less-packets"
 
 To run the test after having run the ``--generate-references``, all that is
 needed is:
 
 .. code-block:: shell
 
-    > python setup.py test --args="--integration=integration.yml -m integration
+    > python setup.py test --args="--integration-tests=integration.yml -m integration
```

Comment:
I think we run this now using `pytest` - have a look in the azure pipelines. Can you change this to using `pytest`

Reply:
Yes, we Astropy moved to `pytest package_name <arguments>`.

Good catch @kaushik94 !

Reply:
@epassaro @wkerzendorf I changed it to `pytest`. However, I don't know if the flags are correct. The tests were not working. I believe I raised an issue a few months ago. Thanks.

Reply:
This one: https://github.com/tardis-sn/tardis/issues/997. If I recall correctly `black_body_sampling` is not a valid parameter anymore in recent TARDIS configuration file schema.

I couldn't reproduce that issue, instead I got: 

```
ERROR  - TypeError: __init__() got an unexpected keyword argument 'self_contained'
``` 

with `pytest tardis --integration-tests=/home/epassaro/Desktop/tardis-sn/tardis/integration.yml -m integration --less-packets --remote-data` and `python setup.py test ...` too. I don't know what happens, the code is a bit old so maybe it's something about newer `pytest` or similar.

About the documentation, seems there are a few typos (quotes misplaced for example).

Reply:
@epassaro yes that's right. @wkerzendorf I think we need to update the yml files for the correct integration tests.

Reply:
@epassaro I think this has something to do with DokuWiki installation, which will post the report to an external server. @wkerzendorf is this correct?